test/integration/application_display_modes_test.rb

```diff
@@ -0,0 +1,114 @@
+require 'test_helper'
+
+class ApplicationDisplayModesTest < ActionDispatch::IntegrationTest
+  include TeamsHelper
+  include Devise::Test::IntegrationHelpers
+
+  def setup
+    @game = create(:game,
+      title: 'Example Game',
+      start: Time.now.utc - 5.days,
+      stop: Time.now.utc + 5.days,
+      description: 'Example Description',
+      terms_of_service: 'Example ToS',
+      terms_and_conditions: 'Example TAC',
+      organization: 'Example Organization',
+      contact_url: 'https://example.com',
+      footer: 'Example Footer',
+      team_size: 10,
+      contact_email: 'example@example.com',
+      open_source_url: 'https://github.com/mitre-cyber-academy/ctf-scoreboard',
+      board_layout: 0
+    )
+  end
+
+  test 'navigation bar shows all fields when logged out' do
+    get "/"
+    
+    assert_select 'a[class=brand]', @game.organization, "Organization should show in navigation bar"
+    assert_select 'a[href=\/game]', I18n.t('application.navbar.challenges'), "Challenges should show in navigation bar"
```

Comment:
Can you see if there is a way to access the route helpers here so we don't have to hardcode all of the `href` paths?